tests/testthat/test-visualizations.R

```diff
@@ -1,5 +1,5 @@
 # Tests for visualization functions
-# library(SingleR); library(testthat); source("setup.R"); source("test-visualizations.R")
+# library(SingleR); library(testthat); source("../GitHub/SingleR/tests/testthat/setup.R"); source("test-visualizations.R")
```

Comment:
Huh? Why?

Reply:
Oh man... never meant to save with that.  Sorry for the sloppy commit!!

Reply:
Fixed